coin-flip/index.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
```

Comment:
This is a best practice with HTML pages. I don't think it'll change things for you, but it's good to have.